### PR TITLE
Added works section and about me section and also styled it to meet requirements

### PR DESCRIPTION
**Pull request by DevaWinner**
This branch contains the further part of the portfolio which contains the works section, and also contains the contact section. It also contains the updated Stylesheet.

### CODE REVIEW

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @DevaWinner , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -24,11 +26,12 @@ <h3><a href="">My Logo</a></h3>
...
-  <!-- Hero Section -->
+  <!--============= Hero Section =====================-->
+
   <main>
-    <div>
-      <h1>Hey there. <br> I'm Alex. </h1>
-      <h1> I'm a software developer</h1>
+    <div class="introduction">
+      <h1 class="intro">Hey there. <br> I'm Winner. </h1>
+      <h1 class="intro1"> I'm a software developer</h1>
```
[the remark this answers is not in the document]
Comment (DevaWinner) [reply index.html:34]: @ademibowale concerning your suggested correction.
1. That part of the code was done on Wednesday and per the instructions, we were to use display: flex for the heading. Display grid is for the Works session, kindly confirm this please.

**Comment by DevaWinner** [comment]
@ademibowale Please can you take a look before I request for a second review?

File: index.html [hunk 2]
```diff
@@ -24,22 +26,240 @@ <h3><a href="">My Logo</a></h3>
...
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>CSS</li>
+              <li>HTML</li>
+            </ul>
+          </div>
+          <div>
+  
+          </div>
+        </div>
```
Comment (AbelG101) [inline index.html:189]: - My apologies for the confusion caused by the previous reviewer as some of the comments weren't related to this milestone. 
- [ ] [OPTIONAL] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul` and not in a `div,` each of them in a `li`. Also, make sure the technology in each category (e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`. I can see you've already implemented it partly but please remove the `div` tags ✔
```
<ul>
      <li>Language</li>
      <ul>
        <li>JavaScript</li>
        <li>Ruby</li>
        <li>HTML</li>
        <li>CSS</li>
      </ul>
      <li>Frameworks</li>
       . . . 
</ul>
```
- [ ] This applies to the list items in the "Frameworks" & "Database Management" as well you'll just have to follow the same principle.
- You can find more [info here](https://www.scaler.com/topics/nested-list-in-html/)

File: index.html [hunk 3]
```diff
@@ -24,22 +26,240 @@ <h3><a href="">My Logo</a></h3>
...
+            <li><a href="https://github.com/DevaWinner"><i class="fa-brands fa-github fa-xl"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/winnera/"><i class="fa-brands fa-linkedin-in fa-xl"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist fa-xl"></i></a></li>
+            <li><a href="https://twitter.com/WinnerDevq"><i class="fa-brands fa-twitter fa-xl"></i></a></li>
+            <li><a href="https://medium.com/@winneraaniekan"><i class="fa-brands fa-medium fa-xl"></i></a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+  </footer>
```
Comment (AbelG101) [inline index.html:263]: - [ ] [OPTIONAL] I like that you're thinking ahead but please make sure to only add features that are part of the current milestone as the contact section is part of next week's requirements. Committing it now will make it difficult for the reviewers next week to give you inline comments so please remove it temporarily and add it next week when you create a new PR. 💪

**Review by AbelG101 — APPROVED** [review]
# Hi @DevaWinner 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**